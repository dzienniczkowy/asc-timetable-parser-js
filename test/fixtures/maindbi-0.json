{
  "r": {
    "type": "maindbi",
    "dbid": "2022",
    "tables": [
      {
        "id": "teachers",
        "data_rows": [
          {
            "id": "-1",
            "short": "Wierzbicka_W"
          },
          {
            "id": "-2",
            "short": "Błaszczyk_A"
          },
          {
            "id": "-3",
            "short": "Bednarek_M"
          }
        ],
        "data_columns": [
          "short"
        ]
      },
      {
        "id": "subjects",
        "data_rows": [
          {
            "id": "-1",
            "name": "wiedza o społeczeństwie",
            "short": "WOS"
          },
          {
            "id": "-2",
            "name": "historia i teraźniejszość",
            "short": "HIT"
          },
          {
            "id": "-3",
            "name": "biologia",
            "short": "biologia"
          }
        ],
        "data_columns": [
          "name",
          "short"
        ]
      },
      {
        "id": "classrooms",
        "data_rows": [
          {
            "id": "-30",
            "short": "Uniwersytet"
          },
          {
            "id": "-31",
            "short": "1"
          },
          {
            "id": "-32",
            "short": "01"
          }
        ],
        "data_columns": [
          "short"
        ]
      },
      {
        "id": "classes",
        "data_rows": [
          {
            "id": "-40",
            "name": "1A",
            "short": "1A"
          },
          {
            "id": "-41",
            "name": "2B - mat. fiz. inf.",
            "short": "2B"
          },
          {
            "id": "-42",
            "name": "3C 3C_integracja",
            "short": "3C"
          }
        ],
        "data_columns": [
          "name",
          "short"
        ]
      },
      {
        "id": "periods",
        "data_rows": [
          {
            "id": "0",
            "name": "0",
            "starttime": "07:10",
            "endtime": "07:55"
          },
          {
            "id": "1",
            "name": "Lekcja 1",
            "starttime": "08:00",
            "endtime": "08:45"
          }
        ],
        "data_columns": [
          "name",
          "starttime",
          "endtime"
        ]
      }
    ]
  }
}
