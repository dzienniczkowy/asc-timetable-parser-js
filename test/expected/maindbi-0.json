{
  "teachers": {
    "-1": {
      "name": "Wierzbicka_W"
    },
    "-2": {
      "name": "Błaszczyk_A"
    },
    "-3": {
      "name": "Bednarek_M"
    }
  },
  "subjects": {
    "-1": {
      "name": "wiedza o społeczeństwie",
      "short": "WOS"
    },
    "-2": {
      "name": "historia i teraźniejszość",
      "short": "HIT"
    },
    "-3": {
      "name": "biologia",
      "short": "biologia"
    }
  },
  "classrooms": {
    "-30": {
      "name": "Uniwersytet"
    },
    "-31": {
      "name": "1"
    },
    "-32": {
      "name": "01"
    }
  },
  "classes": {
    "-40": {
      "name": "1A",
      "short": "1A"
    },
    "-41": {
      "name": "2B - mat. fiz. inf.",
      "short": "2B"
    },
    "-42": {
      "name": "3C 3C_integracja",
      "short": "3C"
    }
  },
  "periods": {
    "0": {
      "name": "0",
      "startTime": "07:10",
      "endTime": "07:55"
    },
    "1": {
      "name": "Lekcja 1",
      "startTime": "08:00",
      "endTime": "08:45"
    }
  }
}
